app.js

```diff
@@ -1,25 +1,59 @@
-const express = require('express')
-const logger = require('morgan')
-const cors = require('cors')
-
-const contactsRouter = require('./routes/api/contacts')
-
-const app = express()
-
-const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short'
-
-app.use(logger(formatsLogger))
-app.use(cors())
-app.use(express.json())
-
-app.use('/api/contacts', contactsRouter)
+const express = require('express');
+const logger = require('morgan');
+const cors = require('cors');
+const { listContacts, getContactById, addContact, removeContact, updateContact } = require('./models/contacts');
+const app = express();
+
+const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short';
+
+app.use(logger(formatsLogger));
+app.use(cors());
+app.use(express.json());
+
+app.get('/api/contacts', async (req, res) => {
```

Comment:
Usuń te metody stąd i napisz tylko 
`const contactsRouter = require('routes/api/contacts');`
`app.use('/api/contacts', contactsRouter)`

Reply:
To sprawi że będzie Ci używać routingu zrobionego w contacts.js

Reply:
poprawione